### ISSUE TITLE
Flex 1.9.0 breaks creating new projects

### ISSUE THREAD

**Issue by nickygerritsen** [symfony/flex#659]
When you create a new project with `symfony new test-project` it now uses Flex 1.9.0 (since that has been released today).
However, running the command fails:

```
nstalling symfony/skeleton (v5.1.99)
  - Installing symfony/skeleton (v5.1.99): Loading from cache
Created project in /Users/nicky/LetsTalk/test-project
Loading composer repositories with package information
Updating dependencies (including require-dev)
Package operations: 1 install, 0 updates, 0 removals
  - Installing symfony/flex (v1.9.0): Loading from cache
Symfony operations: 1 recipe (f3f3dbe996e52949b145f01a5504580d)
  - Configuring symfony/flex (>=1.0): From github.com/symfony/recipes:master
Restricting packages listed in "symfony/symfony" to "5.1.*"

                                                     
  [LogicException]                                   
  No lockfile found. Unable to read locked packages  
                                                     

create-project [-s|--stability STABILITY] [--prefer-source] [--prefer-dist] [--repository REPOSITORY] [--repository-url REPOSITORY-URL] [--add-repository] [--dev] [--no-dev] [--no-custom-installers] [--no-scripts] [--no-progress] [--no-secure-http] [--keep-vcs] [--remove-vcs] [--no-install] [--ignore-platform-reqs] [--] [<package>] [<directory>] [<version>]
                                                                                                                            
  unable to run /usr/local/bin/composer create-project symfony/skeleton /Users/nicky/LetsTalk/test-project : exit status 1  
```

I can also reproduce this by creating an empty directory, placing the skeleton `composer.json` in it, and running `composer install`.

**Comment by nicolas-grekas** [comment]
Fixed in v1.9.1

**Comment by nickygerritsen** [comment]
Fast fix ever 🎉 Thanks!
